lib/perl/Genome/Model/Command/Export/Metadata.pm

```diff
@@ -341,7 +341,7 @@ sub add_to_dump_queue {
     }
 
     if ($obj->isa("Genome::Model::Build")) {
-        $obj->{status} = "Dummy";
+        $obj->{status} = $obj->status || "Dummy";
```

Comment:
`$obj->{status}` should already hold the current value of `$obj->status`, so it should be sufficient to remove this line and the immediately following one that changes `{db_committed}`.